hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/PipelinePlacementPolicy.java

```diff
@@ -413,13 +413,11 @@ private boolean getAnchorAndNextNode(List<DatanodeDetails> healthyNodes,
         removePeers(anchor, healthyNodes);
         mutableExclude.add(anchor);
       } else {
-        LOG.warn("Unable to find healthy node for anchor(first) node.");
+        LOG.debug("Unable to find healthy node for anchor(first) node.");
         throw new SCMException("Unable to find anchor node.",
```

Comment:
I wonder if we should include the entire message in the exception here as well, like in other places. Same comment for the change above this one.

Reply:
Actually, I thought about it a bit more and this change looks good to me as is.